playback/service/src/main/java/de/danoeh/antennapod/playback/service/PlaybackService.java

```diff
@@ -1151,7 +1151,7 @@ private void onPostPlayback(final Playable playable, boolean ended, boolean skip
                     || autoSkipped
                     || (skipped && !UserPreferences.shouldSkipKeepEpisode())) {
                 // only mark the item as played if we're not keeping it anyways
-                DBWriter.markItemPlayed(item, FeedItem.PLAYED, ended || (skipped && almostEnded));
+                DBWriter.markItemPlayed(FeedItem.PLAYED, item);
```

Comment:
I think this is a behavior change. Before, skipping only marked as played when close to the end. Now skipping always marks as played. Is this something we actually want to happen? Related: #7637